test/utils/format.test.js

```diff
@@ -11,15 +11,19 @@ beforeAll(async () => {
 beforeEach(async () => {
 });
 
-*/
+
 test('formatting attonear amounts', async() => {
     /*expect(nearlib.utils.format.formatNearAmount('8999999999837087887')).toEqual('0.000008999999999837087887');
```

Comment:
why are these tests commented out? If they are in yoctonear, makes sense to move to `formatting yoctonear amounts` and use `xtest` to disable for now